.github/workflows/cronjobs.yml

```diff
@@ -6,21 +6,22 @@ on:
   schedule:
     # * is a special character in YAML so you have to quote this string
     - cron: '30 23 * * *' # 11:30 pm
-    - cron: '00 00 * * *' # 12:00 pm
```

Comment:
any fix is onging? maybe It's easier to use `workflow_dispatch ` to test this job manumally.

Reply:
The configuration would not work correctly before. The cronjob will start at 11:30 pm and 12:00 pm, but AutoRebase will skip at 12:00 pm. So it does not work as expected and could be better to remove it.